content/2017-01-08_pyladies-advanced.md

```diff
@@ -0,0 +1,20 @@
+Title: Pyladies Advanced: Srazy pro programováníchtivé dívky 
+Date: 2017-01-08 10:49:00 
+Author: Lenka 'Zouví' Matějová 
+Twitter: Zoellenka
+ Gravatar: zoelles@gmail.com
+About: Zou je holka, co se učí Python. Má stránky [Zoelles](http://zoelles.tumblr.com/), leze po skalách, chodí po slackline a miluje akrojógu.
+
+Od července do půle října 2016 probíhaly různě po Praze v kavárnách, posléze, díky Oskarovi,  v coworkingové zasedačce [Usertech](https://usertechnologies.com/) na Smíchově. Což je super prostor! Děkujeme!
+   
+Celkem proběhlo 22 těchto setkání, které vždy svolala nejlepší organizátorka Veronička.
+Pyladies Advanced bylo maloformátové setkání, nejvíce nás přišlo asi 12, někdy dorazilo i více koučů než Pyladies, ale umožnilo nám to: 
+1) posunout se dále v tom, co děláme. Často jsme díky pomoci koučů vyřešily problém s chybami, na které bychom samy došly jen těžko. Bylo nám vysvětleno spoustu nových témat, jako např. regulární výrazy.  Na tohle na kurzech většinou nezbývá čas a správné programátorce se to hodí znát! 
```

Comment:
... na které bychom samy přišly jen těžko.
... výrazay. ZASE DVĚ MEZERY
... a správné programátorce se to hodí! NEBO ... správná programátorka by to měla znát!